client/utils/rest.go

```diff
@@ -1,5 +1,6 @@
 package utils
 
+import "C"
```

Comment:
This seems to have broken cross compilation between Darwin and Linux. Can we safely remove this line or is there any reason why we shouldn't do so?

Reply:
If we want this cross-compilation ability can we test for it in CI? Then we'll know in the future if we break it.

Reply:
I'd second the idea, though we have to figure out how go about it - https://github.com/golang/go/issues/19470